CMakeLists.txt

```diff
@@ -122,19 +122,20 @@ if 'real_prefix' in dir(sys):
   endif ()
 
   # boost-python
-  set (BOOST_PYTHON_COMPONENT_NAMES
-       python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}
-       python${PYTHON_VERSION_MAJOR} python)
-  foreach (NAME ${BOOST_PYTHON_COMPONENT_NAMES})
-    if (NOT Boost_FOUND)
-      message (STATUS "Looking for component ${NAME}")
-      find_package (Boost QUIET COMPONENTS ${NAME})
-    endif ()
-  endforeach ()
+  #set (BOOST_PYTHON_COMPONENT_NAMES
+  #     libboost-python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}
+  #     python${PYTHON_VERSION_MAJOR} python)
+  #foreach (NAME ${BOOST_PYTHON_COMPONENT_NAMES})
+  #  if (NOT Boost_FOUND)
+  #    message (STATUS "Looking for component ${NAME}")
+  #    find_package (Boost QUIET COMPONENTS ${NAME})
+  #  endif ()
+  #endforeach ()
+  FIND_PACKAGE(Boost COMPONENTS python)
```

Comment:
No idea why I did not use this, this is way better, just remove the messy old comments :+1: